src/cppapi/position.cpp

```diff
@@ -0,0 +1,62 @@
+
+#include "openmc/position.h"
+#include "openmc/cppapi/position.h"
+#include "openmc/capi.h"
+#include "openmc/error.h"
+
+#include <pybind11/operators.h>
+
+#include <stdexcept>
+#include <vector>
+
+
+namespace openmc {
+
+using std::vector;
+namespace py = pybind11;
+
+//==============================================================================
+// CPP API
+//==============================================================================
+//Add elements of Position to the module
+void init_position(py::module& m)
+{
+  py::class_<Position> (m,"Position")
+    .def(py::init<double, double, double>())
+    .def(py::init([](std::tuple<double,double,double> tup) {
```

Comment:
Added this constructor for direct conversion from tuple. Needs to be expanded to implicitly transform ints